fre/pp/configure_script_yaml.py

```diff
@@ -30,165 +31,94 @@ def yamlInfo(yamlfile,experiment,platform,target):
-                  elif "land" in compvalue:
-                    f.write("inputRealm=land\n")
-                  elif "river" in compvalue:
-                    f.write("inputRealm=land\n")
-                  elif "ocean" in compvalue:
-                    f.write("inputRealm=ocean\n")
-                  elif "aerosol" or "tracer" in compvalue:
-                    f.write("inputRealm=atmos\n")
-                elif compkey == "sourceGrid": 
-                  f.write(f"inputGrid={compvalue}\n")
-                elif compkey == "interpMethod":
-                  f.write(f"{compkey}={compvalue}\n")
-                elif compkey == "sources":
-                  f.write(f"{compkey}={compvalue} ")
-                  try: 
-                    i["timeSeries"]
-                    for elem in i['timeSeries']:
-                      for key,value in elem.items():
-                        if key == "source":
-                          f.write(f"{value} ")
-                  except:
-                    print("No timeseries information")
-
-                  f.write("\n")
-
-                if compkey == "xyInterp" and compvalue == y["defaultxyInterp"]:
-                  f.write(f"outputGridType=default\n")
-                elif compkey == "xyInterp": 
-                  gridLat=compvalue.split(",")[0]
-                  gridLon=compvalue.split(",")[1]
-
-                  f.write(f"outputGridLat={gridLat}\n")
-                  f.write(f"outputGridLon={gridLon}\n")
-                  f.write(f"outputGridType={gridLat}_{gridLon}\n") 
+      for i in value:
+        comp = i.get('type')
+        sources = i.get('sources')
+        interp_method = i.get('interpMethod')
+
+        # set remap items
+        rose_remap.set(keys=[f'{comp}', 'sources'], value=f'{sources}')
+        # if xyInterp doesnt exist, grid is native
+        if i.get("xyInterp") == None:
+          rose_remap.set(keys=[f'{comp}', 'grid'], value='native')
+        # if xyInterp exists, component can be regridded
+        else:
+          interp_split = i.get('xyInterp').split(',')
+          rose_remap.set(keys=[f'{comp}', 'grid'], value=f'regrid-xy/{interp_split[0]}_{interp_split[1]}.{interp_method}')
+
+        # set regrid items
+        if i.get("xyInterp") != None:
+          rose_regrid.set(keys=[f'{comp}', 'sources'], value=f'{sources}')
+          rose_regrid.set(keys=[f'{comp}', 'inputRealm'], value=f'{i.get("inputRealm")}')
+          rose_regrid.set(keys=[f'{comp}', 'inputGrid'], value=f'{i.get("sourceGrid")}')
+          rose_regrid.set(keys=[f'{comp}', 'interpMethod'], value=f'{interp_method}')
+          interp_split = i.get('xyInterp').split(',')
+          rose_regrid.set(keys=[f'{comp}', 'outputGridLon'], value=f'{interp_split[1]}')
+          rose_regrid.set(keys=[f'{comp}', 'outputGridLat'], value=f'{interp_split[0]}')
+          rose_regrid.set(keys=[f'{comp}', 'outputGridType'], value=f'{interp_split[0]}_{interp_split[1]}.{interp_method}')
```

Comment:
Just double checking, so for the outputGridType, the output would look like this:
```outputGridType=180_288.conserve_order2```

Instead of previously, like this, for example:
```outputGridType=180_288```

Similar to `grid` comment

Reply:
Yep. those have to match until we can retire the rose app configs.. and then have those two apps use the pp.yaml directly.